packages/rlc-common/src/metadata/buildSampleEnvFile.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { RLCModel } from "../interfaces.js";
+
+const sampleEnvText = `
+# App registration secret for AAD authentication
+AZURE_CLIENT_SECRET=
+AZURE_CLIENT_ID=
+AZURE_TENANT_ID= 
+`;
+
+export function buildSampleEnvFile(model: RLCModel) {
```

Comment:
Could you share the failure message with me if no sample.env provided?

Reply:
![image](https://github.com/user-attachments/assets/5fb12e04-eaec-4854-99b8-3290e6df3bfb)